study.md

```diff
@@ -40,14 +40,17 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+if you type a string, all lower case and use .capitalize it will capitalize the first letter
```

Comment:
In this case it does, but we want to know what `!` means in general when it's in the name of a method.